scripts/UpdateRepos.ps1

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env powershell
+
+<#
+.SYNOPSIS
+    Updates each repo Universe builds to new dependencies.props
+.PARAMETER Source
+    The NuGet package source to find the lineup on.
+.PARAMETER LineupID
+    The ID of the Lineup to determine which versions to use.
+.PARAMETER LineupVersion
+    The version of the Lineup to be used.
+#>
+[cmdletbinding(SupportsShouldProcess = $true)]
+param(
+    [Parameter(Mandatory=$true)]
+    [string]$Source,
+    [Parameter(Mandatory=$true)]
+    [string]$LineupID,
+    [Parameter(Mandatory=$true)]
+    [string]$LineupVersion
+)
+
+$ErrorActionPreference = 'Stop'
+Set-StrictMode -Version 2
+
+Import-Module "$PSScriptRoot/common.psm1" -Scope Local -Force
+
+$RepoRoot = Resolve-Path "$PSScriptRoot\.."
+$ModuleDirectory = Join-Path $RepoRoot "modules"
+
+try {
+    Push-Location $ModuleDirectory
```

Comment:
Lift this above the 'try'. If it were to fail, the Pop-Location would move you to the wrong place.